website/blog/2021-06-24-introduce-superbench.md

```diff
@@ -8,4 +8,113 @@ author_url: https://github.com/TobeyQin
 tags: [superbench, announcement]
 ---
 
-We are very happy to introduce [SuperBench](https://github.com/microsoft/superbenchmark) to help you validate AI infrastructure.
+
+This blog is to introduce [SuperBench](https://github.com/microsoft/superbenchmark) to help you validate AI infrastructure.
+
+We designed and developed SuperBench as a avalidation and profiling tool for AI infrastructure. 
+We try our best to create a modular, easy-to-use, standardized, and extensible tool. 
+Help users to complete and accurate assessment of AI infrastructure, and bring good user experience.
+
+
+## The Advantages of SuperBench
+
+### Easy-to-use CLI
+
+In order to achieve one-click deployment, one-click user experience, SuperBench provides a command line interface to help users deploy and run benchmarks. 
+Empowered by SuperBench CLI, user can deploy and run their benchmarks with only one command, which greatly shorten the learning time of using tools, 
```

Comment:
`which greatly shorten the learning time`

Reply:
fixed